lib/spack/spack/platforms/cray_xc.py

```diff
@@ -50,9 +51,9 @@ def setup_platform_environment(self, pkg, env):
             similar to linux/standard linker behavior
         """
         env.set('CRAYPE_LINK_TYPE', 'dynamic')
-        cray_wrapper_names = join_path(spack.build_env_path, 'cray')
-        if os.path.isdir(cray_wrapper_names):
-            env.prepend_path('PATH', cray_wrapper_names)
+#        cray_wrapper_names = join_path(spack.build_env_path, 'cray')
+#        if os.path.isdir(cray_wrapper_names):
+#            env.prepend_path('PATH', cray_wrapper_names)
```

Comment:
I would delete this instead of commenting it out. You don't need to worry about losing the old code, that's the whole point of version control.